deb/openmediavault/debian/openmediavault.dirs

```diff
@@ -6,6 +6,7 @@ usr/share/php/openmediavault
 var/lib/openmediavault
 var/lib/openmediavault/apt
 var/lib/openmediavault/cron.d
+var/lib/openmediavault/dashboard.d
```

Comment:
I am presuming that adding this means that the directory will get automatically created?

For my local dev, i manually created the folder

Reply:
Confirmed this appears to b the case